pages/eth-unit-conversion.test.tsx

```diff
@@ -7,16 +7,16 @@ describe(EthUnitConversion.name, () => {
   it('sets wei field and gets a correct value in the rest of fields', async () => {
     const root = render(<EthUnitConversion />);
 
-    const weiField = (await root.findByLabelText('wei')) as HTMLInputElement;
+    const weiField = (await root.findByLabelText(/^wei/)) as HTMLInputElement;
 
     fireEvent.change(weiField, {
       target: { value: '10000000001000002333343' },
     });
 
     expect(weiField.value).toEqual('10000000001000002333343');
 
-    const gweiField = (await root.findByLabelText('gwei')) as HTMLInputElement;
-    const ethField = (await root.findByLabelText('eth')) as HTMLInputElement;
+    const gweiField = (await root.findByLabelText(/gwei/)) as HTMLInputElement;
```

Comment:
Why do we want to use regexes where not necessary? To stay consistent with the assertions above? If so - shouldn't we change that in the whole project?

Reply:
I moved the label a few levels up in the DOM so the hitbox for mouse clicks is more forgiving. The exact text inside the label might be something like "wei 10^-18" now. We also don’t care about casing here I think.